src/components/VoteRow.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+const Wrapper = styled.div`
+  width: 150px;
+  margin: 10px;
+`;
+const Button = styled.button`
+  width: 100px;
+`;
+
+export default function VoteRow({ candidate, order, setCandidates }) {
+  const { id } = candidate;
+  const vote = async () => {
```

Comment:
이벤트 핸들러 네이밍은 이렇게 하는게 더 좋을 것 같습니다 👍 

```suggestion
  const handleVoteCount= async () => {
```